### PR TITLE
Player test

### PR DESCRIPTION
**Pull request by claudds**
with @lori-dalkin 

Added tests for the player class, including mocks for various classes which Player depends on, and stubs for methods called in these classes.

This addresses both parts of A1 part 2, both in breaking dependencies and in fixing the problem of not enough testing being applied to the project.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -181,8 +181,12 @@ lazy val user = module("user", Seq(common, memo, db, hub, rating)).settings(
     )
 )
 
-lazy val game = module("game", Seq(common, memo, db, hub, user, chat)).settings(
+/*lazy val game = module("game", Seq(common, memo, db, hub, user, chat)).settings(
```
Comment (jacobrs) [inline build.sbt]: is this commented out module necessary?

File: modules/game/src/test/PlayerTest.scala [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  val mockColor = mock(classOf[Color])
+
+  //User class
+  val mockUser = mock(classOf[User])
+
+  //Glicko class
+  var mockGlicko = mock(classOf[Glicko])
+
+  //Perf class
+  var mockPerf = mock(classOf[lila.rating.Perf])
```
Comment (jacobrs) [inline modules/game/src/test/PlayerTest.scala]: Lot's of `var` in the code, these are mutable variables and are usually frowned upon in Scala. You should consider switching them to `val` which might involve refactoring the code a bit. Immutability is an important aspect of functional programming.

File: project/Dependencies.scala [hunk 3]
```diff
@@ -12,6 +12,7 @@ object Dependencies {
     val lilaMaven = "lila-maven" at "https://raw.githubusercontent.com/ornicar/lila-maven/master"
     val prismic = "Prismic.io kits" at "https://s3.amazonaws.com/prismic-maven-kits/repository/maven/"
 
+
```
Comment (jacobrs) [inline project/Dependencies.scala]: ?

File: build.sbt (hunk 1 above)
Comment (lori-dalkin) [reply build.sbt]: It's not, I had put it when I wasnt sure what I was changing. Will remove.

File: project/Dependencies.scala (hunk 3 above)
Comment (lori-dalkin) [reply project/Dependencies.scala]: Left from when something else was there that I didnt need in the end. Will remove.

**Review by jacobrs — APPROVED** [review]

**Review by BDSchoeler — APPROVED** [review]

**Pull request merged**